x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -199,7 +197,7 @@ const UnstyledProcessEventDot = React.memo(
           })
         | null;
     } = React.createRef();
-    const { colorMap, cubeAssetsForNode } = useResolverTheme();
+    const { colorMap, cubeAssetsForNode } = useResolverTheme(resolverComponentInstanceID);
```

Comment:
:grey_question: I may have missed this, but why are we passing in a component instance ID to the theming hook? You won't have different themes for different Resolvers (AFAIK), right? But this makes it look like you could.

Reply:
just to have stable (non-random) html IDs for symbols